requirements/dev.txt

```diff
@@ -14,7 +14,7 @@ parameterized~=0.7.4
 # Integration tests
 click~=7.1
 dateparser~=0.7
-boto3~=1.17
+boto3>=1.23,<2
```

Comment:
Why we needed boto version bump here? In order to support a new parameter to `lambda_client.invoke`?

Reply:
It was a miss from the FURL PR actually, as the FURL integration test is using `lambda_client.get_function_url_config`, which is only available in boto3 1.22.15 or above